Command/RepositoryCrawlerCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ConnectionsBundle\Command;
+
+use ONGR\ElasticsearchBundle\Command\AbstractElasticsearchCommand;
+use ONGR\ConnectionsBundle\Crawler\RepositoryCrawler;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Repository crawler.
+ */
+class RepositoryCrawlerCommand extends AbstractStartServiceCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct()
+    {
+        parent::__construct('ongr:repository-crawler:crawl', 'Repository crawler.');
+
+        $this
+            ->addOption(
+                'action-name',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Set specific pipeline event name (see documentation for details)',
+                null
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $start = microtime(true);
+
+        /** @var RepositoryCrawler $repositoryCrawler */
+        $repositoryCrawler = $this->getContainer()->get('ongr_connections.repository_crawler_service');
+        $repositoryCrawler->setOutput($output);
+        $actionName = $input->getOption('action-name');
+
+        if ($actionName != null) {
+            $repositoryCrawler->setTarget($actionName);
+        }
+
+        $repositoryCrawler->startCrawler();
+
+        $output->writeln('');
```

Comment:
The following lines are implemented in `CommandBenchmark` class, please use it instead.